nodes/selection/ScSelectVertexByConnections.py

```diff
@@ -0,0 +1,89 @@
+import bpy
+import bmesh
+from bpy.props import IntProperty, BoolProperty
+from bpy.types import Node
+from .._base.node_base import ScNode
+from .._base.node_selection import ScSelectionNode
+
+
+class ScSelectVertexByConnections(Node, ScSelectionNode):
+    bl_idname = "ScSelectVertexByConnections"
+    bl_label = "Select Vertex By Connections"
+
+    in_connections: IntProperty(default=1, min=0, update=ScNode.update_value)
+    in_extend: BoolProperty(default=True, update=ScNode.update_value)
+
+    def init(self, context):
+        super().init(context)
+        self.inputs.new("ScNodeSocketNumber", "Connections").init("in_connections", True)
+        self.inputs.new("ScNodeSocketBool", "Extend").init("in_extend")
+        # work in vertex mode:
+        # With this it is not necessary to manually force the mode to vertices mode:
+        self.in_selection_type = {'VERT'}
+        self.in_extend = False
+
+    def error_condition(self):
+        return (
+                super().error_condition()
+                or int(self.inputs["Connections"].default_value) < 0
+                # prevent other modes:
+                or self.in_selection_type != {'VERT'}
```

Comment:
Since the `in_selection_type` is an input socket which can contain multiple values (set), the correct condition would be:
`or (not 'VERT' in self.inputs["Selection Type"].default_value)`